virttest/qemu_vm.py

```diff
@@ -3751,7 +3751,7 @@ def migrate(self, timeout=virt_vm.BaseVM.MIGRATE_TIMEOUT, protocol="tcp",
                 remote_port=None, not_wait_for_migration=False,
                 fd_src=None, fd_dst=None, migration_exec_cmd_src=None,
                 migration_exec_cmd_dst=None, env=None,
-                migrate_capabilities=None):
+                migrate_capabilities=None, mig_inner_funcs=None):
```

Comment:
Having free-form `mig_inner_funcs` is great and flexible, but I'd suggest either adding built-ins for most common functions or simply hardcoding post-copy migration to avoid custom per-test implementation. (more about that here: https://github.com/dagrh/tp-qemu/commit/07beb5733e6d921ab3df82bafb1fc168029e2fbf#commitcomment-28669032 )